selfdrive/modeld/modeld.cc

```diff
@@ -14,11 +13,11 @@
 
 ExitHandler do_exit;
 // globals
-bool run_model;
+bool live_calib_seen;
 mat3 cur_transform;
-pthread_mutex_t transform_lock;
+std::mutex transform_lock;
 
-void* live_thread(void *arg) {
+void live_thread() {
```

Comment:
Guess we should rename this to `calibration_thread` since that's what it's doing.